vale/Vocab/CockroachDB/accept.txt

```diff
@@ -23,17 +17,10 @@ autovacuum
 avro
 AZ
 AZs
```

Comment:
Could this and the preceding be combined into a regex?

Reply:
I was going to work on that in more detail in a separate PR, but I've fixed that line specifically for this PR.